src/common/import_session.c

```diff
@@ -322,7 +322,7 @@ static const char *_import_session_path(struct dt_import_session_t *self, gboole
     return NULL;
   }
 
-  new_path = dt_variables_expand(self->vp, pattern, FALSE);
+  new_path = dt_util_normalize_path(dt_variables_expand(self->vp, pattern, FALSE));
```

Comment:
This leaks memory and you do not free the now inaccessible char* returned by dt_variables_expand.

Reply:
PR no longer uses `dt_util_normalize_path()` now